lib/server.js

```diff
@@ -102,9 +102,16 @@ module.exports = function(aOptions) {
       serverType.createServer.apply(serverType, serverParams).
         listen(serverPort);
 
+      // Azure App Service ports are actually pipes like "\\.\pipe\3212d91e-bc1a-4720-8792-49cf5f9eba86"
+      let serverPortRaw = String.raw`${options.serverPort}`;
```

Comment:
Use `const` here and on the following line instead of `let`.

Reply:
used const instead of let